WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -127,12 +116,69 @@ import CoreTelephony
 
 private extension ZendeskUtils {
 
-    func enableZendesk(_ enabled: Bool) {
-        ZendeskUtils.zendeskEnabled = enabled
+    static func enableZendesk(_ enabled: Bool) {
```

Comment:
Nipticky suggestion!

`enableZendesk(false)` actually disables Zendesk, but that's not clear by just looking at the method signature.

How about.... 

- `static func toggleZendesk(enabled: Bool)` 
- `static func configureZendesk(enabled: Bool)`
- `static func enableZendesk()` + `static fund disableZendesk()`